libraries/adaptive-expressions/src/expressionFunctions.ts

```diff
@@ -2988,7 +2983,37 @@ export class ExpressionFunctions {
                         return {value, error};
                     }),
                 ReturnType.Boolean,
-                ExpressionFunctions.validateIsMatch)
+                ExpressionFunctions.validateIsMatch),
+            
+            // Type Checking Functions
+            new ExpressionEvaluator(ExpressionType.isString, ExpressionFunctions.apply(
+                (args: any[]): boolean =>  typeof args[0] === 'string'),
+            ReturnType.Boolean,
+            ExpressionFunctions.validateUnary),
```

Comment:
not just `Unary`, the param's return type should be `string` or `object`.
we can consider to use 
```
validateUnaryString
```

Reply:
> not just `Unary`, the param's return type should be `string` or `object`.
> we can consider to use
> 
> ```
> validateUnaryString
> ```

My understanding is isString() can take any type argument, and check if the type of args[0] is 'string'. If validateUnaryString used here, once a number is parsed to the function. It will throw an error instead of return false.

Reply:
ok make sense